src/components/Sidebar/Sidebar.test.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
I'm not sure. Do you want to avoid flow check for test files? I think `// @flow` is just missing here.

I think we should also use flow in our tests or maybe we should ignore them in the flow configuration if we don't need type checking.

Reply:
Looks like only `platform.service.test.js` and `migrations.test.js` have `// @flow`, rest are disabled. Maybe using Flow in the test files is not necessary?